src/containers/paper-canvas.jsx

```diff
@@ -82,14 +82,36 @@ class PaperCanvas extends React.Component {
         }
     }
     convertToBitmap () {
-        this.props.clearSelectedItems();
-        const raster = paper.project.activeLayer.rasterize(72, false /* insert */);
-        raster.onLoad = function () {
-            const subCanvas = raster.canvas;
-            getRaster().drawImage(subCanvas, raster.bounds.topLeft);
-            paper.project.activeLayer.removeChildren();
-            this.props.onUpdateSvg();
+        // @todo if the active layer contains only rasters, drawing them directly to the raster layer
+        // would be more efficient.
+        // Export svg
+        const guideLayers = hideGuideLayers(true /* includeRaster */);
+        const bounds = paper.project.activeLayer.bounds;
+        const svg = paper.project.exportSVG({
+            bounds: 'content',
+            matrix: new paper.Matrix().translate(-bounds.x, -bounds.y)
+        });
+        showGuideLayers(guideLayers);
+        
+        // Get rid of anti-aliasing
+        // @todo get crisp text?
+        svg.setAttribute('shape-rendering', 'optimizeSpeed');
+        const svgString = (new XMLSerializer()).serializeToString(svg);
+
+        // Put anti-aliased SVG into image, and dump image back into canvas
+        const img = new Image();
+        img.onload = function () {
+            const raster = new paper.Raster(img);
+            raster.onLoad = function () {
+                const subCanvas = raster.canvas;
+                getRaster().drawImage(
+                    subCanvas,
+                    new paper.Point(Math.floor(bounds.topLeft.x), Math.floor(bounds.topLeft.y)));
+                paper.project.activeLayer.removeChildren();
+                this.props.onUpdateSvg();
+            }.bind(this);
         }.bind(this);
+        img.src = `data:image/svg+xml;charset=utf-8,${svgString}`;
```

Comment:
Wow. That's a little bit hilarious, all written out!

It does seem like you could simplify it a bit by replacing the `.bind(this)` stuff with arrow functions; is there a reason not to do so?

Reply:
Sounds good. I am just not used to the es6 notation still.